WordPress/gutenberg
# Add command: Manage categories while Editing a category template

## What?
Enhancement for the **Editing a category template**  mentioned in https://github.com/WordPress/gutenberg/issues/50407

## Why?
Currently there is no command to manage categories while editing category templates.

## How?
This PR adds adds "Manage Categories" command by default when the command palette is opened while editing category templates.

## Testing Instructions

- Log in to your WordPress dashboard.
- Navigate to the Site Editor.
- Click on Templates from the left panel.
- The main section will display a list of all available templates.
- Select the "All Archives" template, or any Category or Tag template you wish to edit.
- The editor panel for the selected template will open.
- Open the Command Palette by either pressing Cmd + K (on macOS) or using the command palette icon at the top.
- The "Manage Categories" or "Manage Tags" command will appear by default.
- This command will not appear for any other templates other than category or tag templates.
- Click on the displayed command to be redirected to the respective management page (Categories or Tags) in the WordPress dashboard.


## Screenshots or screencast 

https://github.com/user-attachments/assets/b24f70a7-37d6-465e-a03b-92a672571e1b
